sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks);net47</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS8032</NoWarn>
```

Comment:
1. I don't think it is correct to suppress 'CS8032: analyzer cannot be created after changing PackageReference'. Can we fix the underlying issue.
2. Any project-specific change is better made locally. Changes here impact all dependent projects.

Reply:
I hit this error when using VS 2019. But this error is gone when I switched to VS2022. I suppose only VS2022 is supported?